databricks/koalas/series.py

```diff
@@ -4898,6 +4898,8 @@ def __getattr__(self, item: str_type) -> Any:
                 return property_or_func.fget(self)  # type: ignore
             else:
                 return partial(property_or_func, self)
+        elif not hasattr(ks.Series, item):
+            raise AttributeError("'Series' object has no attribute '{}'".format(item))
         return self.getField(item)
```

Comment:
I think you can remove this. We don't currently support nested structure in Series. Just directly raise `raise AttributeError("'Series' object has no attribute '{}'".format(item))`.

Reply:
@HyukjinKwon Thanks! 👍